powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/i18n/LocalizationProvider.java

```diff
@@ -87,6 +88,27 @@ public GenericServiceException buildExceptionForCode(String code, Locale locale)
         return new GenericServiceException(code, message, localizedMessage);
     }
 
+    /**
+     * Build rollbacking exception for given error code in English.
```

Comment:
This is correct, but we also should update the documentation for a regular "build" methods, that produce non-rollbacking exceptions. I would simply update the following methods, that exception produced is "non-rollbacking" one:
```java
public GenericServiceException buildExceptionForCode(String code) { ... }
public GenericServiceException buildExceptionForCode(String code, Locale locale) { ... }
```